components/tidb_query_vec_aggr/src/impl_max_min.rs

```diff
@@ -64,6 +65,7 @@ impl<T: Extremum> super::AggrDefinitionParser for AggrFnDefinitionParserExtremum
 
         let out_ft = aggr_def.take_field_type();
         let out_et = box_try!(EvalType::try_from(out_ft.as_accessor().tp()));
+        let out_coll = out_ft.as_accessor().collation();
```

Comment:
Maybe we can leave the error case here and don't need to manually check for it:

```rust
let out_coll = out_ft.as_accessor().collation()?;
```

or if the above not work:

```rust
let out_coll = box_try!(out_ft.as_accessor().collation());
```